src/styles/Input.scss

```diff
@@ -54,6 +86,15 @@ label{
     size: 14px;
 }
 
+span{
```

Comment:
Can we avoid overriding of direct HTML elements ? and if it is required can we move this to index.SCSS file so that CSS for span tag will be same ?